### DISCUSSION TITLE
Product related maintenance mode

### DISCUSSION THREAD

**Discussion by finnjosten** [Paymenter/Paymenter#294] [Ideas]
As a admin you can flip a button to start a maintenance mode on your products.
You can then also add a post behind it to show what you will be doing and stuff (kinda like announcements) or you can attach a link to it for an external page.

This either shows for all customers or as a admin you can choice which products will have maintenance. Then if a client as that product they will get the banner (if you dont have the product you wont) and if people plan on buying it they will get a warning when they buy it.

Maybe even add a start and end option to choose when the banner will show and when not. The banner will be placed maybe under the navbar and if you click it you go to the post or to the link you set in the admin page for this.

**Comment by CorwinDev** [comment]
How would that work with configurable options? eg on checkout someone choosed a different location then the default

**Comment by finnjosten** [comment]
It can either work for product wide so no matter the configurable options that product will just have a warning, and the warning will most likely contain some text in which you can maybe give some info about which location/version or try to redirect users to the post liked to it in which you can put info about which locations and products and all exactly.
^^
This could maybe even linked to something like uptime kuma or hetrix so it will automaticly sync not sure how just an idea

or say you have configurable options setup an admin can choose which location/version gets it and it will then show when a user selects that location/version, but if the product doesn't configurable options have any it will just show up

Keep in mind you can also just make a banner under the navbar that just shows if their is maintenance instead of a specific product
